$OpenBSD: patch-modules_gui_qt4_dialogs_messages_cpp,v 1.2 2017/05/27 08:40:14 ajacoutot Exp $
Use qt4 atomics there, they're portable (taken from FreeBSD)

Index: modules/gui/qt4/dialogs/messages.cpp
--- modules/gui/qt4/dialogs/messages.cpp.orig
+++ modules/gui/qt4/dialogs/messages.cpp
@@ -145,7 +145,7 @@ MessagesDialog::~MessagesDialog()
 
 void MessagesDialog::changeVerbosity( int i_verbosity )
 {
-    atomic_store( &this->verbosity, i_verbosity );
+    verbosity = i_verbosity;
 }
 
 void MessagesDialog::updateConfig()
@@ -339,7 +339,7 @@ void MessagesDialog::MsgCallback( void *self, int type
 {
     MessagesDialog *dialog = (MessagesDialog *)self;
     char *str;
-    int verbosity = atomic_load( &dialog->verbosity );
+    int verbosity = dialog->verbosity;
 
     if( verbosity < 0 || verbosity < (type - VLC_MSG_ERR)
      || unlikely(vasprintf( &str, format, ap ) == -1) )
